receiver/otlpreceiver/config.go

```diff
@@ -66,6 +81,20 @@ func (cfg *Config) Unmarshal(conf *confmap.Conf) error {
 
 	if !conf.IsSet(protoHTTP) {
 		cfg.HTTP = nil
+	} else {
+		// Verify URL path sanity
+		signalURLPaths := []*string{&cfg.HTTP.TracesURLPath, &cfg.HTTP.MetricsURLPath, &cfg.HTTP.LogsURLPath}
```

Comment:
It's probably a matter of taste, but I would the sanitization on its own function, and call it three times. Like this (untested):

```go
var err error

if cfg.HTTP.TracesURLPath, err = sanitizeURLPath(cfg.HTTP.TracesURLPath); err != nil {
  return err
}

if cfg.HTTP.MetricsURLPath, err = sanitizeURLPath(cfg.HTTP.MetricsURLPath); err != nil {
  return err
}

if cfg.HTTP.LogsURLPath, err = sanitizeURLPath(cfg.HTTP.LogsURLPath); err != nil {
  return err
}

func sanitizeURLPath(urlPath string) string {
  u, err := url.Parse(*urlPath)
  if err != nil { 
    return fmt.Errorf("invalid HTTP URL path set for signal: %w", err)
  }

  if !path.IsAbs(u.Path) {
    u.Path = "/" + u.Path
  }

  return u.Path
}

```